packages/date-picker/src/panel/date.vue

```diff
@@ -97,23 +105,23 @@
               :default-value="defaultValue ? new Date(defaultValue) : null"
               :date="date"
               :cell-class-name="cellClassName"
-              :disabled-date="disabledDate">
+              :disabled-date="disabledDate || (minimum || maximum ? dateOutOfRange : '')">
```

Comment:
If `disabledDate` is not defined, but `minimum` or `maximum` is, then use a pre-defined `dateOutOfRange` function